### PR TITLE
LPS-190975 Create Output Validation Type Selector

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPS-190975

Hey @naraandrade, to better test this code you can use the following mock right below the line [142 on EditObjectValidation.tsx](https://github.com/liferay-objects/liferay-portal/pull/2371/files#diff-42c0300001f8fd2c57ccc5a7e6a185c157a3e3d2f3d305493bddaa85c03e9730) and replace where the validationResponseJSON is being used by the mock:

You can replace the `id`, `name` and `value` for an existing field in your environment

```tsx
const responseMock: ObjectValidation = {
	...validationResponseJSON,
	objectValidationRuleSettings: [
		{
			id: 33143,
			name: 'birthday',
			value: '958f2dc5-ece4-9cfe-5714-d251f04a16b1',
		},
	],
};
```

### CODE REVIEW

**Comment by naraandrade** [comment]
Just started reviewing :)

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
Just started reviewing :)

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/FieldBase.scss [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 .lfr-objects__field-base {
 	&-tooltip-icon {
-		color: $secondary-l1;
+		color: #6b6c7e;
```
Comment (naraandrade) [inline modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/FieldBase.scss:5]: @danvb112 You can use the variable $gray-600 for this color.

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Radio/Radio.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+		alignPosition?: 'top' | 'bottom';
+		content?: string;
+		disableScroll?: boolean;
+		header?: string;
+	};
+	required?: boolean;
+	tooltip?: string;
+}
+
+export function Radio({
```
Comment (naraandrade) [inline modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Radio/Radio.tsx:41]: @danvb112 shouldn't we call it RadioButton/RadioField? I'm not sure if Radio is enough to describe this component.

**Review by naraandrade — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Radio/Radio.tsx (hunk 2 above)
Comment (danvb112) [reply modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Radio/Radio.tsx]: Thanks for the sugestion! I put the folder name and component name as RadioField to follow React Conventions

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:sf

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/FieldBase.scss [hunk 3]
```diff
@@ -2,6 +2,6 @@
 
 .lfr-objects__field-base {
 	&-tooltip-icon {
-		color: #6b6c7e;
+		color: $gray-600;
```
Comment (naraandrade) [inline modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/FieldBase.scss:5]: Hi @danvb112, please squash this change and the ones about the RadioField to the previous commits. This helps during the review and also if we need to revert commits in the future. Thanks!

Comment (danvb112) [reply modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/FieldBase.scss:5]: Good catch, Done! Thanks @naraandrade

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Review by naraandrade — APPROVED** [review]

**Comment by danvb112** [comment]
Sent to: https://github.com/brianchandotcom/liferay-portal/pull/138466

**Pull request closed without merging**